sdk/core/core-http/src/policies/deserializationPolicy.ts

```diff
@@ -270,7 +270,7 @@ function parse(
   xmlContentTypes: string[],
   operationResponse: HttpOperationResponse
 ): Promise<HttpOperationResponse> {
-  const errorHandler = (err: Error & { code: string }) => {
+  const errorHandler = (err: Error & { code: string }): any => {
```

Comment:
I think the return should be `Promise<unknown>` or `Promise<any>` if that doesn't work

Reply:
Sounds good!

Reply:
Promise<unknown> didn't work, I'll use Promise<any>